src/components/Sponsor/Section/Sponsors.styles.ts

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 1rem;
+  gap: 1rem;
+`;
+
+export const Title = styled.h1`
+  font-size: 2rem;
+  text-align: center;
+  margin: 1rem;
+`;
```

Comment:
Because we reuse this component in every section I think it is best if we move it to the globalStyles file with this style:

```suggestion
export const SectionTitle = styled.h2`
  font-size: 1.75rem;
  text-align: center;
`;
```